src/zope/i18n/compile.py

```diff
@@ -1,54 +1,61 @@
-from contextlib import closing
+
 import logging
-import os
-from os.path import join
-from stat import ST_MTIME
+logger = logging.getLogger('zope.i18n')
+
+def _cannot_compile_mo_file(domain, lc_messages_path):
+    logger.critical("Unable to compile messages: Python `gettext` library missing.")
```

Comment:
This message makes me think it's talking about https://docs.python.org/3/library/gettext.html, when it's actually talking about https://pypi.python.org/pypi/python-gettext.